core/ui/src/main/java/com/mashup/core/ui/widget/MashupToolbar.kt

```diff
@@ -31,10 +32,11 @@ fun MashUpToolbar(
     modifier: Modifier = Modifier,
     title: String,
     showBackButton: Boolean = false,
-    showCloseButton: Boolean = false,
+    showActionButton: Boolean = false,
     showBottomDivider: Boolean = false,
     onClickBackButton: () -> Unit = {},
-    onClickCloseButton: () -> Unit = {}
+    onClickActionButton: () -> Unit = {},
+    @DrawableRes actionButtonDrawableRes: Int? = null,
```

Comment:
```suggestion
    @DrawableRes actionButtonDrawableRes: Int = R.drawable.ic_close,
```

디폴트값으로 null이 아닌 close를 주는 것도 좋을 것 같아요! 아래 비지니스 로직을 파악하지 않고도 매개변수로 null이면 close icon이 보이겠구나라고 확인하기 쉬울 것 같은 개인적인 느낌이에요!

Reply:
오오 안그래도 함수 내부에서 널 체크 후 close 아이콘 설정해주는게 이상했었는데 훨씬 좋네용 👍✨
수정했습니당 9f0be95c6dafacb80d7f5af243051ecf2ace268f